tests/validate/test_scenario_test.py

```diff
@@ -54,43 +55,92 @@ def test_scenario_test_get_dataset_items(
     test_slice,
     slice_items,
 ):
+    # Arrange
```

Comment:
Refactored the `test_scenario_test_get_*` methods via [AAA-C](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) bc of my OCD

Reply:
Feel free to drop the link next to one of the comments 👍

Reply:
I think it's ok, pretty common paradigm and future tests can use the structure of these functions as inspiration. Don't feel too fond about linking to Medium lmao